lib/app_server.py

```diff
@@ -224,8 +224,7 @@ def kill():
     def find_exe(cls, builddir):
         cls.builddir = builddir
         cls.binary = TarantoolServer.binary
-        cls.debug = bool(re.findall(r'-Debug', str(cls.version()),
-                                    re.I))
+        cls.debug = bool(re.findall(r'^Target:.*-Debug$', str(cls.version())))
```

Comment:
I search the web around, whether `CMAKE_BUILD_TYPE` should be interpreted case sensitive or case insensitive and found that the latter seems to be considered as a kind of the right way. For example, LLVM:

https://github.com/llvm/llvm-project/blob/699396131f745ec8b42ae33cb0e4553a783b6674/llvm/CMakeLists.txt#L371-L376
https://lists.llvm.org/pipermail/llvm-commits/Week-of-Mon-20150810/292471.html

I guess we don't handle it correctly in tarantool, but we likely should (sooner or later). I think we should reflect it in https://github.com/tarantool/tarantool/issues/3686.

I propose to leave the interpretation as case insensitive in test-run to don't change it twice forth and back.

Reply:
Okay, left the regular expression case-insensitive.